shell-script/public-gateway-checker.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Should be `/usr/bin/env bash` here instead to ensure portability.

Reply:
OK